lib/presentation/utils/image_picker_helper.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:permission_handler/permission_handler.dart';
+import 'package:trip_tally/presentation/theme/app_dimensions.dart';
+import 'package:trip_tally/presentation/utils/enums/context_extensions.dart';
+
+class ImagePickerHelper {
+  const ImagePickerHelper();
```

Comment:
As it has only static method, we can `prevent instantiation` by adding private constructor